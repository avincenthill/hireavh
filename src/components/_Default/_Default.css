._default-container {
  background-color: var(--c-4);
  display: flex;
  flex-direction: column;
  justify-content: center;
  width: 90vw;
  height: 80vh;
  margin-bottom: var(--s-m);
}

._default-paragraph {
  margin: var(--s-s);
}
