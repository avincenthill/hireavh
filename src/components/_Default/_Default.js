import './_Default.css';
import { NavList } from 'components/Nav/Nav';
import Page from 'components/Page/Page';
import React from 'react';
// import content from "content/content";

class _Default extends React.Component {
  render() {
    return (
      <Page>
        <div className="_default-container">
          <h1 className="_default-title">_Default Title</h1>
          <h2 className="_default-subtitle">_Default Subtitle</h2>
          <p className="_default-paragraph">
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis a
            porta nisi. Mauris auctor porttitor risus non hendrerit. Etiam
            pulvinar vitae eros eu sodales. Etiam rhoncus mollis blandit.
            Curabitur at mattis mi, vitae posuere massa. Lorem ipsum dolor sit
            amet, consectetur adipiscing elit.
          </p>
        </div>
        <NavList></NavList>
        <hr></hr>
      </Page>
    );
  }
}

export default _Default;
