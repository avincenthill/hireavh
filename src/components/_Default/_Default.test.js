import React from 'react';
import _Default from './_Default';
import renderer from 'react-test-renderer';

describe('_Default', () => {
  describe('snapshot', () => {
    it('should match the last saved snapshot', () => {
      const tree = renderer.create(<_Default />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
