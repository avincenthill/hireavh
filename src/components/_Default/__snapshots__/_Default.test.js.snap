// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`_Default snapshot should match the last saved snapshot 1`] = `
<div
  className="page"
>
  <div>
    <div
      className="nav nav-top"
    >
      <div
        className="nav-icon $
          this.props.isBottom ? \\"nav-icon-bottom\\" : \\"nav-icon-top\\"
        }"
      >
        <svg
          fill="currentColor"
          height="1em"
          onClick={[Function]}
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": "var(--c-1)",
              "height": "3rem",
              "width": "3rem",
            }
          }
          viewBox="0 0 8 16"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M8 4v1H0V4h8zM0 8h8V7H0v1zm0 3h8v-1H0v1z"
            fillRule="evenodd"
          />
        </svg>
      </div>
    </div>
    <a
      className="header-container"
      href="/about"
    >
      <div
        className="header-switch-theme"
      >
        <div
          className="toggle-switch"
        >
          <input
            className="toggle-switch-checkbox"
            defaultChecked={true}
            id="toggle-switch-theme"
            onChange={[Function]}
            type="checkbox"
          />
          <label
            className="toggle-switch-label"
            htmlFor="toggle-switch-theme"
          >
            <span
              className="toggle-switch-inner"
              data-no="🌙"
              data-yes="☀️"
            />
          </label>
        </div>
      </div>
      🏠 about
    </a>
  </div>
  <div
    className="page-content-container"
  >
    <div
      className="_default-container"
    >
      <h1
        className="_default-title"
      >
        _Default Title
      </h1>
      <h2
        className="_default-subtitle"
      >
        _Default Subtitle
      </h2>
      <p
        className="_default-paragraph"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis a porta nisi. Mauris auctor porttitor risus non hendrerit. Etiam pulvinar vitae eros eu sodales. Etiam rhoncus mollis blandit. Curabitur at mattis mi, vitae posuere massa. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
      </p>
    </div>
    <div
      className="nav-list"
    >
      <div>
        <h2
          className="link"
        >
          <a
            className="link-a"
            href="/about"
          >
            <span>
              🏠 about
            </span>
          </a>
        </h2>
      </div>
      <div>
        <h2
          className="link"
        >
          <a
            className="link-a"
            href="/blog"
          >
            <span>
              ✍️ blog
            </span>
          </a>
        </h2>
      </div>
      <div>
        <h2
          className="link"
        >
          <a
            className="link-a"
            href="/projects"
          >
            <span>
              📦 projects
            </span>
          </a>
        </h2>
      </div>
      <div>
        <h2
          className="link"
        >
          <a
            className="link-a"
            href="/resume"
          >
            <span>
              📄 resume
            </span>
          </a>
        </h2>
      </div>
      <div>
        <h2
          className="link"
        >
          <a
            className="link-a"
            href="/contact"
          >
            <span>
              📇 contact
            </span>
          </a>
        </h2>
      </div>
    </div>
    <hr />
  </div>
  <div>
    <div
      className="nav nav-bottom"
    >
      <div
        className="nav-icon $
          this.props.isBottom ? \\"nav-icon-bottom\\" : \\"nav-icon-top\\"
        }"
      >
        <svg
          fill="currentColor"
          height="1em"
          onClick={[Function]}
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": "var(--c-1)",
              "height": "3rem",
              "width": "3rem",
            }
          }
          viewBox="0 0 8 16"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M8 4v1H0V4h8zM0 8h8V7H0v1zm0 3h8v-1H0v1z"
            fillRule="evenodd"
          />
        </svg>
      </div>
    </div>
    <a
      className="footer-container"
      href="/about"
    >
      <span>
        🏠 about
      </span>
    </a>
  </div>
</div>
`;
